index.html

```diff
@@ -1,3 +1,130 @@
-<html>
-<body>Gazr is coming ...</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Gazr [gather] - Opinionated way to launch common goals on projects</title>
```

Comment:
launch common goals ?

Reply:
I'd prefer "launch common tasks", it seems better to me.

Reply:
Fixed